editor/src/clj/editor/protobuf.clj

```diff
@@ -374,11 +373,11 @@ Macros currently mean no foreseeable performance gain, however."
                  (cond
                    (resource-field? field-info)
                    (case (:field-rule field-info)
-                     :repeated [ [[key]] ]
-                     :required [ [key] ]
+                     :repeated [ [[key]]]
+                     :required [ [key]]
                      :optional (if-some [field-default (declared-default class key)]
-                                 [ [{key field-default}] ]
-                                 [ [key] ]))
+                                 [ [{key field-default}]]
+                                 [ [key]]))
```

Comment:
Hmm, these nested vectors got a bit messed up.

Reply:
It's only training space removed by parinfer... I'll remove leading space too for symmetry.